packages/ui/src/components/Typography/Typography.module.css

```diff
@@ -0,0 +1,230 @@
+.rootNormalize {
+  margin: 0;
+  padding: 0;
+  display: block;
+}
+
+.rootItalic {
+  font-style: italic;
+}
+
+.root b,
+.rootBold.rootBold.rootBold {
+  font-weight: var(--ui_font_weight_bold);
+}
+
+.rootBlack {
```

Comment:
jeez, can we simplify these styles somehow? Let's do a smarter approach with some sort of single map here rather than a lot copy-pasting (I mean code duplication). I'll look at it once again later to provide more specific thoughts about that

Reply:
Кажется css решения будут всегда производительнее, gzip и brotli хорошо ужимают повторяющиеся части